3ds Max/Max2Babylon/Exporter/BabylonExporter.Mesh.cs

```diff
@@ -1097,12 +1097,16 @@ int CreateGlobalVertex(IIGameMesh mesh, BabylonAbstractMesh babylonAbstractMesh,
             // tangent
             if (exportParameters.exportTangents)
             {
-                int indexTangentBinormal = mesh.GetFaceVertexTangentBinormal(face.MeshFaceIndex, facePart, 1);
-                IPoint3 normal = vertex.Normal.Normalize;
-                IPoint3 tangent = mesh.GetTangent(indexTangentBinormal, 1).Normalize;
-                IPoint3 bitangent = mesh.GetBinormal(indexTangentBinormal, 1).Normalize;
-                int w = GetW(normal, tangent, bitangent);
-                vertex.Tangent = new float[] { tangent.X, tangent.Y, tangent.Z, w };
+                int mapChannel = 1; // Texture Coordinates
+                if (mesh.GetNumberOfTangents(mapChannel) != 0)
+                {
+                    int indexTangentBinormal = mesh.GetFaceVertexTangentBinormal(face.MeshFaceIndex, facePart, mapChannel);
+                    IPoint3 normal = vertex.Normal.Normalize;
+                    IPoint3 tangent = mesh.GetTangent(indexTangentBinormal, mapChannel).Normalize;
+                    IPoint3 bitangent = mesh.GetBinormal(indexTangentBinormal, mapChannel).Normalize;
+                    float w = GetW(normal, tangent, bitangent);
+                    vertex.Tangent = new float[] { tangent.X, tangent.Y, tangent.Z, -1 };
```

Comment:
why is vertex.Tangent.w always -1 now?

Reply:
Well ... because i commit the wrong version ... won the🥇